validator/client/validator_log.go

```diff
@@ -29,7 +29,7 @@ func (v *validator) LogAttestationsSubmitted() {
 			"TargetRoot":        fmt.Sprintf("%#x", bytesutil.Trunc(attLog.data.Target.Root)),
 			"AttesterIndices":   attLog.attesterIndices,
 			"AggregatorIndices": attLog.aggregatorIndices,
-		}).Info("Submitted new attestations")
```

Comment:
Submitted new Attestations is right. It composes more than one index more than one validator here.

Reply:
Fixed!